deploy/charts/istio-csr/values.yaml

```diff
@@ -99,6 +99,10 @@ app:
       # -- Container port to serve istio-csr gRPC service.
       port: 6443
 
+  istio:
+    # -- The istio revisions that are currently installed in the cluster.
+    revisions: []
```

Comment:
Suggestion: I guess typically deploying a new revision would be a day 2 operation and the users would likely first deploy Istio with a single control plane and at some point later decide to deploy a new revision.
So, it's likely that `istio-csr` would first be installed without `revisions` explicitly set. Then if they wanted to add a new revision 'canary', they would actually need to set this value to `["canary","default"] else the cert would only be created for `istiod-canary.istio-system.svc` DNS name.

Would it make sense to explicitly default `revisions` to ["default"]?

Reply:
+1 Great point. I think that's a good default

Reply:
Yes, I thought about this however "default" is actually a [special revision case](https://istio.io/latest/docs/setup/upgrade/canary/#default-tag) which is different from not setting a revision.

It is my understanding that one need to actually setup their revision from `nothing` -> `default` -> `default + foo`. That assumption might be wrong though.